pkg/runtime/bindings/registry.go

```diff
@@ -35,44 +35,44 @@ type Registry interface {
 	CreateInputBinding(name string) (bindings.InputBinding, error)
 }
 
-type bindingsRegistry struct {
+type BindingsRegistry struct {
```

Comment:
Coud u revert it to lowercase, `bindingsRegistry`

Reply:
Done @seeflood